aten/src/ATen/native/mkldnn/xpu/detail/Utils.cpp

```diff
@@ -1,5 +1,8 @@
+#include <ATen/Context.h>
 #include <ATen/native/ConvUtils.h>
 #include <ATen/native/mkldnn/xpu/detail/Utils.h>
+#include <dnnl.hpp>
+#include <dnnl_common.hpp>
```

Comment:
@ZhiweiYan-96 this file `dnnl_common.hpp` is in the `oneapi/dnnl` subdirectory of `intel/oneapi/dnnl/latest/include/` which is not included by default in the oneapi include paths. On windows this is causing a problem. Can we update the path here to include `oneapi/dnnl` prefix? Do you want me to file an issue?